src/features/Settings/components/AddOrEditNetworkNode.tsx

```diff
@@ -318,7 +318,11 @@ export default function AddOrEditNetworkNode({
                   },
                   shouldEstimateGasPrice: false,
                   color: undefined,
-                  selectedNode: node.name
+                  selectedNode: node.name,
+                  blockExplorer: makeExplorer({
+                    name: networkName!,
+                    origin: ETHPLORER_URL
```

Comment:
This wont be valid for custom networks

Reply:
It seemed to be the default fallback.
Should we add a field to customise the explorer ?

Reply:
Hmm, probably not necessary for a dev feature though